SWLOR.Game.Server/Feature/AIDefinition/GenericAIDefinition.cs

```diff
@@ -45,39 +45,39 @@ static float CalculateAverageHP(uint creature)
             }
 
             // Force Heal
-            if (CheckIfCanUseFeat(self, self, FeatType.ForceHeal5, () => hpPercentage <= 80 && activeConcentration == FeatType.Invalid))
+            if (CheckIfCanUseFeat(self, self, FeatType.ForceHeal5, () => hpPercentage <= 99 && activeConcentration == FeatType.Invalid))
```

Comment:
Is the change to 99 expected? This wasn't in the original commit

Reply:
Yes sorry should have mentioned that. Basically it's so they start trying to get this regen up ASAP once combat starts. Based on what I observed they'll use it too slowly otherwise.